src/online/unigrid.jl

```diff
@@ -38,3 +38,10 @@ function updateZ!(
     end
     return Z
 end
+
+function updateZ(
+    rng::AbstractRNG, Z::AbstractVector, alg::UniGrid, X::AbstractVector
+)
+    Zn = deepcopy(Z)
+    return updateZ!(rng, Zn, alg, X)
+end
```

Comment:
This could probably set up as a generic function instead.

Reply:
Yeah, that is probably a good idea, but I need to figure out how to pass `kwargs...` through a function. I did it once, and now I can't remember how it goes.